### ISSUE TITLE
Text formatting shortcuts

### ISSUE THREAD

**Issue by ElbertCode** [flxzt/rnote#587]
**Is your feature request related to a problem? Please describe.**
If you select a text and try to use the common shortcut for a text formatting (e. g. Ctrl + B to make i bold), the selected text will be replaced by the letter you typed (b in the example).

**Describe the solution you'd like**
For the available text formating options in Rnote I would suggest these shortcuts:
| Shortcut | Action |
|--------|--------|
| Ctrl + B | Bold |
| Ctrl + I | Italic |
| Ctrl + U | Underline |
| Ctrl + L | Align Left |
| Ctrl + E | Align Center |
| Ctrl + R | Align Right |
| Ctrl + J | Justified |

**Describe alternatives you've considered**
Maybe these shortcuts could also be localized depending on the language as for example the shortcut in german to make a text bold is usually Ctrl + Shift + F.

**Comment by mohamedAdhamc** [comment]
I think this probably the most frustrating thing I've had while taking notes for the past few months. Anyone did/planning to do it ? If not I think I could personally add them, and make a PR.

**Comment by Kneemund** [comment]
It should be fairly simple to add them, I think the main problem is figuring out what key combinations to use. CTRL + I is already used for importing documents, so we either need to change that, or use e.g. CTRL + SHIFT as modifiers for these.

**Comment by mohamedAdhamc** [comment]
> It should be fairly simple to add them (the same way CTRL + A is implemented), I think the main problem is figuring out what key combinations to use. 

How about just making them customizable ? Because, generally it's hard to reach an agreement on such shortcuts.

**Comment by Kneemund** [comment]
Yes, but we still need sensible defaults. Making shortcuts customizable is tracked in #590, and I don't think anyone has tackled that so far.
